pkg/docker/config/config.go

```diff
@@ -52,6 +52,10 @@ var (
 	ErrNotSupported = errors.New("not supported")
 )
 
+func newAuthPathDefault(path string) authPath {
```

Comment:
Non-blocking: Moving the `authPath` type just above this constructor would be a tiny bit easier to follow.